staging/src/k8s.io/api/events/v1/types.go

```diff
@@ -27,7 +27,8 @@ import (
 // Event is a report of an event somewhere in the cluster. It generally denotes some state change in the system.
 type Event struct {
 	metav1.TypeMeta `json:",inline"`
-	// +optional
+
+	// required.
```

Comment:
Just adding "required" as a comment does not actually make it appear as required in the openapi schema. You actually have to remove the `+optional` comment and `omitempty` tag to do that.

Reply:
Oh alright. Thanks for the heads up. Fixing this